### ISSUE TITLE
Unable to log into Github Copilot in Visual Studio Code

### ISSUE THREAD

**Issue by marisamacias** [Microsoft/vscode-tools-for-ai#2591]
## Expected Behavior
Login successful

## Actual Behavior
"Failed to sign into Github. Would you like to try again?"

## Steps to Reproduce the Problem

  1. Open visual studio code
  2. Click in the lower right corner "Sign in to use Copilot"
  3. Wait

## Specifications

  - Version: 0.103
  - Platform: Windows 11
  - Screen Reader: na

**Comment by Siglud** [comment]
Please refer to this post to find the solution about your issue: https://github.com/orgs/community/discussions/151649

And please note, this is a repo for Azure Machine Learning extension for VS Code, not the repo for GitHub Copilot.

If you have any question about GitHub copilot, please create post in [community](https://github.com/orgs/community/discussions/) or raise an issue in [VSCode's repo](https://github.com/microsoft/vscode/issues)
